mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -204,6 +204,11 @@ void assertNotDead() {
       this.parent = parent;
     }
 
+    /**
+     * Makes this service available via the new scope's {@link MortarScope#findService}
+     * method. If the service implements {@link Scoped} it will be registered with
+     * the new scope.
+     */
     public Builder withService(String serviceName, Object service) {
```

Comment:
I wasn't a fan of the hidden behavior in register() is the Scoped was also a Bundle... compile time APIs are much clearer and safe than javadoc.